src/components/NoteManager/components/NotesActions.tsx

```diff
@@ -26,10 +27,15 @@ export function NotesActions() {
     handleRedo,
     handleImport,
     handleExport,
+    handleDeleteNotes,
     handleSetRemoteSources,
   } = useContext(NotesContext) as INotesContext;
 
+  const [confirmButtonTimeoutId, setConfirmButtonTimeoutId] = useState<number | null>(null);
+  const [secondsLeft, setSecondsLeft] = useState(3);
   const [isModalOpen, setModalOpen] = useState(false);
+  const [confirmDelete, setConfirmDelete] = useState(false);
+  const [confirmDeleteDisabled, setConfirmDeleteDisabled] = useState(false);
```

Comment:
```suggestion
  const confirmDeleteDisabled = confirmDelete && secondsLeft > 0;
```

nit: This state entry is redundant and could be replaced with something like this.

Reply:
thank you for good suggestion